scripts/deploy_assisted_service.sh

```diff
@@ -16,6 +16,13 @@ export OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE:-}
 export ENABLE_KUBE_API=${ENABLE_KUBE_API:-false}
 export ENABLE_KUBE_API_CMD="ENABLE_KUBE_API=${ENABLE_KUBE_API}"
 
+# If auth type isn't specified and we're using kube-api, use local
+if [ "$ENABLE_KUBE_API" == "true" -a "$AUTH_TYPE" == "" ]; then
```

Comment:
we want to always use local auth on kube-api?
it that's true, I think we have a place where we set all kind of things related to kube-api settings. So put it there

Reply:
I did it this way so that it was still configurable if someone wanted to set it back to none. But I might be able to do the same elsewhere.

Reply:
> I think we have a place where we set all kind of things related to kube-api settings

Maybe this? https://github.com/openshift/assisted-test-infra/blob/c9beeddc7597138d54c89c8e5608305ec7cd81e8/Makefile#L113-L117

Reply:
yes